tritium-metrics/src/main/java/com/palantir/tritium/metrics/MetricRegistries.java

```diff
@@ -204,6 +207,29 @@ static void registerCache(MetricRegistry registry, Cache<?, ?> cache, String nam
                 .forEach((key, value) -> registerWithReplacement(registry, key, value));
     }
 
+    public static ScheduledExecutorService instrument(
```

Comment:
We should javadoc these. It would be helpful to document all the expected metric names + types as a reference for dashboard authors.

Reply:
Added JavaDoc to the public methods